feat: add set_ignore_cursor_events (#1884)
rust-windowing/winit: CHANGELOG.md

- On Web, added support for `DeviceEvent::MouseMotion` to listen for relative mouse movements.
- Added `Window::drag_window`. Implemented on Windows, macOS, X11 and Wayland.
- On X11, bump `mio` to 0.7.

> - On macOS and Windows, added `set_ignore_cursor_events` to make window catch or ignore mouse events.

Reviewer: ```suggestion
- On macOS and Windows, added `set_ignore_cursor_events` to let the window ignore mouse events.
```
